tests/test_translation.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+
+from nlptest import Harness
+
+
+class TranslationTestCase(unittest.TestCase):
+
+    def setUp(self) -> None:
+        self.harness = Harness(
+            task="translation",
+            model='t5-base',
+            hub="huggingface",
+            data="Translation-test"
+        )
```

Comment:
We can do this in a next sprint while tackling #565 but it would be great to have this test running for all third party libraries